queries/kusto/highlights.scm

```diff
@@ -0,0 +1,62 @@
+(source) @variable
+(let_statement
+  (let_keyword) @keyword
+  (identifier) @variable)
+
+(function_call
+  (identifier) @function.call)
+[
+  (type_cast_function)
+  (to_scalar_function)
+  (between_function)
+  (datatable_function)
+] @function.call
+
+(typed_parameter
+  (identifier) @parameter
+  )
+
+(function_arguments (identifier) @parameter)
+(operation (identifier) @variable)
+(compound_expression (identifier) @variable)
+(binary_expression (identifier) @variable)
+(assignment (identifier) @variable)
+(property_identifier (identifier) @variable)
+(property_index (identifier) @variable)
+(sort_by (identifier) @variable)
+(range_operation (identifier) @variable)
```

Comment:
With these many `(identifier) @variable`, I suggest using one `(identifier) @variable` at the top then keeping those that capture not as variable below it

Reply:
Done